src/libs/IOUUtils.js

```diff
@@ -36,12 +45,17 @@ function calculateAmount(participants, total, isDefaultUser = false) {
  * @param {Object} iouReport
  * @param {String} actorEmail
  * @param {Number} amount
+ * @param {String} currency
  * @param {String} type
  * @returns {Object}
  */
-function updateIOUOwnerAndTotal(iouReport, actorEmail, amount, type = CONST.IOU.REPORT_ACTION_TYPE.CREATE) {
+function updateIOUOwnerAndTotal(iouReport, actorEmail, amount, currency, type = CONST.IOU.REPORT_ACTION_TYPE.CREATE) {
     const iouReportUpdate = {...iouReport};
 
+    if (currency && currency !== iouReport.currency && isNetworkOffline) {
```

Comment:
We shouldn't try to update the total if we don't have a currency, right?
```suggestion
    if (currency !== iouReport.currency && isNetworkOffline) {
```